csharp-style.md

```diff
@@ -0,0 +1,477 @@
+# C# at Google Style Guide
+
+This style guide is for C# code developed internally at Google, and is the
+default style for C# code at Google. It makes stylistic choices that conform to
+other languages at Google, such as Google C++ style and Google Java style.
+
+## Formatting guidelines
+
+### Naming rules
+
+Naming rules follow
+[Microsoft's C# naming guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/naming-guidelines).
+Where Microsoft's naming guidelines are unspecified (e.g. private and local
+variables), rules are taken from the
+[CoreFX C# coding guidelines](https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/coding-style.md)
+
+Rule summary:
+
+#### Code
+
+*   Names of classes, methods, enumerations, public fields, public properties,
+    namespaces: `PascalCase`.
+*   Names of local variables, parameters: `camelCase`.
+*   Names of private, protected, internal and protected internal fields and
+    properties: `_camelCase`.
+*   Naming convention is unaffected by modifiers such as const, static,
+    readonly, etc.
+*   For casing, a "word" is anything written without internal spaces, including
+    acronyms. For example, `MyRpc` instead of ~~`MyRPC`~~.
+*   Names of interfaces start with `I`, e.g. `IInterface`.
+
+#### Files
+
+*   Filenames and directory names are `PascalCase`, e.g. `MyFile.cs`.
+*   Where possible the file name should be the same as the name of the main
+    class in the file, e.g. `MyClass.cs`.
+*   In general, prefer one core class per file.
+
+### Organization
+
+*   Modifiers occur in the following order: `public protected internal private
+    new abstract virtual override sealed static readonly extern unsafe volatile
+    async`.
+*   Namespace `using` declarations go at the top, before any namespaces. 'using'
```

Comment:
Should the second using be in backticks?

Reply:
Fixed and updated.